### DISCUSSION TITLE
1.18 snapshot fabric loader not working

### DISCUSSION THREAD

**Discussion by UrModaClaped** [FabricMC/fabric-api#1662] [Player support]
when itry to open any of the the fabric 1.18 snapshot loaders it just has a red error box on the top right corner of minecraft launcher.

any fix?
![image](https://user-images.githubusercontent.com/80205253/130476988-b73c0add-f08f-4109-8e20-0cd2c10e8df9.png)

**Comment by modmuss50** [comment]
Have you installed and ran the experimental snapshot without fabric? Fabric will not install the custom profile for you.
